src/main/java/pl/pop/interview/master/question/Question.java

```diff
@@ -20,9 +18,9 @@ public class Question {
     @GeneratedValue
     private Long id;
     private String content;
-    private String correctAnswer;
+    private boolean correctAnswer;
```

Comment:
Tutaj, też bym pomyślał nad tym czy na pewno "correctAnswer" jest potrzebne, skoro to jest pytanie i raczej "answer" by samo w sobie informowało, że jest to poprawna odpowiedź.